android/modules/ui/src/java/ti/modules/titanium/ui/TextAreaProxy.java

```diff
@@ -58,6 +60,8 @@ public TextAreaProxy()
 	{
 		super();
 		defaultValues.put(TiC.PROPERTY_VALUE, "");
+		defaultValues.put(TiC.PROPERTY_LINES, 1);
```

Comment:
I think `lines` should default to -1.

For backward compatibility, we should respect the TextArea's `height` property. The `lines` property is used to set the height of the TextArea by line height instead (definitely a good feature) but the developer should opt-in to it.